lightning/src/ln/chan_utils.rs

```diff
@@ -479,31 +484,31 @@ pub(crate) fn get_htlc_redeemscript_with_explicit_keys(htlc: &HTLCOutputInCommit
 	}
 }
 
-/// note here that 'a_revocation_key' is generated using b_revocation_basepoint and a's
+/// note here that 'revocation_key' is generated using remote_revocation_basepoint and broadcaster's
 /// commitment secret. 'htlc' does *not* need to have its previous_output_index filled.
 #[inline]
 pub fn get_htlc_redeemscript(htlc: &HTLCOutputInCommitment, keys: &TxCreationKeys) -> Script {
-	get_htlc_redeemscript_with_explicit_keys(htlc, &keys.a_htlc_key, &keys.b_htlc_key, &keys.revocation_key)
+	get_htlc_redeemscript_with_explicit_keys(htlc, &keys.broadcaster_htlc_key, &keys.countersignatory_htlc_key, &keys.revocation_key)
 }
 
 /// Gets the redeemscript for a funding output from the two funding public keys.
 /// Note that the order of funding public keys does not matter.
-pub fn make_funding_redeemscript(a: &PublicKey, b: &PublicKey) -> Script {
-	let our_funding_key = a.serialize();
-	let their_funding_key = b.serialize();
+pub fn make_funding_redeemscript(broadcaster: &PublicKey, countersignatory: &PublicKey) -> Script {
+	let broadcaster_funding_key = broadcaster.serialize();
+	let countersignatory_funding_key = countersignatory.serialize();
 
 	let builder = Builder::new().push_opcode(opcodes::all::OP_PUSHNUM_2);
-	if our_funding_key[..] < their_funding_key[..] {
-		builder.push_slice(&our_funding_key)
-			.push_slice(&their_funding_key)
+	if broadcaster_funding_key[..] < countersignatory_funding_key[..] {
+		builder.push_slice(&broadcaster_funding_key)
+			.push_slice(&countersignatory_funding_key)
 	} else {
-		builder.push_slice(&their_funding_key)
-			.push_slice(&our_funding_key)
+		builder.push_slice(&countersignatory_funding_key)
+			.push_slice(&broadcaster_funding_key)
 	}.push_opcode(opcodes::all::OP_PUSHNUM_2).push_opcode(opcodes::all::OP_CHECKMULTISIG).into_script()
 }
 
 /// panics if htlc.transaction_output_index.is_none()!
-pub fn build_htlc_transaction(prev_hash: &Txid, feerate_per_kw: u32, to_self_delay: u16, htlc: &HTLCOutputInCommitment, a_delayed_payment_key: &PublicKey, revocation_key: &PublicKey) -> Transaction {
+pub fn build_htlc_transaction(prev_hash: &Txid, feerate_per_kw: u32, contest_delay: u16, htlc: &HTLCOutputInCommitment, delayed_payment_key: &PublicKey, revocation_key: &PublicKey) -> Transaction {
```

Comment:
broadcaster_delayed_payment_key, I believe?

Reply:
See point above. We should scope `revocation_key` or let it as it.